stdlib/sqlite3/dbapi2.pyi

```diff
@@ -233,16 +233,16 @@ class ProgrammingError(DatabaseError): ...
 class Row:
     def __init__(self, *args: Any, **kwargs: Any) -> None: ...
     def keys(self): ...
-    def __eq__(self, other): ...
-    def __ge__(self, other): ...
-    def __getitem__(self, index): ...
-    def __gt__(self, other): ...
+    def __eq__(self, __other): ...
+    def __ge__(self, __other): ...
```

Comment:
Arguably this shouldn't exist. The type defines the tp_richcompare slot, which I believe automatically gets mapped to all six Python-level dunders, but it always returns NotImplemented for anything other than EQ and NE. As a result, `sorted(list[Row])` doesn't work, but a type checker will allow it. Anyway, that's a problem for another day.